docs/file-request.md

```diff
@@ -0,0 +1,70 @@
+File Requests
+=============
+
+File request objects represent a file request associated with a folder.
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Get a File Request's Information](#get-a-file-requests-information)
+- [Copy a File Request's Information](#copy-a-file-requests-information)
+- [Update a File Request's Information](#update-a-file-requests-information)
+- [Delete a File Request](#delete-a-file-request)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+Get a File Request's Information
+------------------------
+
+Calling `FileRequestsManager.GetFileRequestByIdAsync(string fileRequestId)`returns information on a file request.
+
```

Comment:
Maybe we could add sample tags to this method and all others in this file?

Like this:

`<!-- sample get_file_requests_id -->`

Then this sample will be automatically placed in API reference.

Reply:
Done!